### ISSUE TITLE
ignores in package.json not recognized properly

### ISSUE THREAD

**Issue by florianb** [sindresorhus/atom-linter-xo#22]
It seems that the ignores in the `package.json` aren't recognized properly:

```
  "xo": {
    "esnext": true,
    "globals": [
      "atom"
    ],
    "ignores": [
      "atom/**",
      "spec/**"
    ]
  }
```

Leads to the following output on command line (cwd = project-root):

```
$ xo
$
```

But the xo-linter still reports errors. I could provide a screenshot, but i guess that's not necessary.

**Comment by sindresorhus** [comment]
Probably related to https://github.com/sindresorhus/atom-linter-xo/issues/19. I'm looking into it.

**Comment by florianb** [comment]
Thanks - i will try to help.. It depends on my 📆 for the next 🌞

**Comment by marionebl** [comment]
I dug a into this and think it is more of an [xo](https://github.com/sindresorhus/xo) issue. linter-xo has to use the `lintText` method, which does not respect the `ignores` option.

**Comment by marionebl** [comment]
This should be fixed for installs using xo `>=0.17`. Feel free to reopen if the problem persists.
